src/Hyperledger.Aries.TestHarness/Hyperledger.Aries.TestHarness.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <IsPackable>false</IsPackable>
+      <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
This is a packable project, should target netstandard.

Reply:
If it is a packable project then why IsPackable set to false or you mean somthing else by packable?
fluent validation 9.x.x does not work with netstandard anymore so you can not upgrade to ver 9.x.x.

Reply:
Hyperledger.Aries.AspNetCore removed from Hyperledger.Aries.TestHarness as it is not used.